resources/looking-glass-client.desktop.in

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Type=Application
+Version=1.0
```

Comment:
This looks good, but the version should reflect the current version of the build as such this file needs to be generated by the build process.

Reply:
No.

Here Version= refers to version of the XDG Desktop Entry spec and the value should be 1.0. 

https://specifications.freedesktop.org/desktop-entry-spec/1.1/recognized-keys.html

Reply:
Oh great! thanks for clarifying.